digdag-standards/src/main/java/io/digdag/standards/command/DockerCommandExecutor.java

```diff
@@ -70,6 +70,9 @@ private Process startWithDocker(Path projectPath, TaskRequest request, ProcessBu
         }
         else {
             imageName = baseImageName;
+            if (dockerConfig.has("pull-always")) {
```

Comment:
Thank you for review.
 
I think this options similar to `build` option.
If a user unnecessary pull image before run, the user just does not define the option.

But this is just my opinion so i would' like to change implementation along your advice if digdag team want.